.changeset/wise-pigs-push.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Update Accordion styling
```

Comment:
Worth to talk about border bottom -> top, and new `status` prop?

Reply:
I did the further updates quite quickly, so this PR will receive some more polish e.g. adding a QA story to have visual regression, updating the docs and adding a story for status.